curriculum/challenges/english/20-upcoming-python/learn-recursion-by-solving-the-tower-of-hanoi-puzzle/650c6082e5586f9e3acfcd3b.md

```diff
@@ -0,0 +1,72 @@
+---
+id: 650c6082e5586f9e3acfcd3b
+title: Step 52
+challengeType: 20
+dashedName: step-52
+---
+
+# --description--
+
+Now, delete the `rods` dictionary and turn its keys into the variables `A`, `B`, and `C`, respectively, keeping their values. Refactor your code to reflect these changes. If you see the output on the terminal, you correctly did this step. 
+
+# --hints--
+
+You should turn `rods['A']` into a variable named `A` keeping its value.
+
+```js
+({ test: () => assert.match(code, /A\s*=\s*list\s*\(\s*range\s*\(\s*NUMBER_OF_DISKS\s*,\s*0\s*,\s*-\s*1\s*\)\s*\)/) })
+```
+
+You should turn `rods['B']` into a variable named `B` keeping its value.
+
+```js
+({ test: () => assert.match(code, /B\s*=\s*\[\s*\]/) })
+```
+
+You should turn `rods['C']` into a variable named `C` keeping its value.
+
+```js
+({ test: () => assert.match(code, /C\s*=\s*\[\s*\]/) })
+```
+
+You should refactor your code after deleting the `rods` object.
+
+```js
+({ test: () => {
+  assert.match(code, /print\s*\(\s*A\s*,\s*B\s*,\s*C\s*,\s*('|")\\n\1\s*\)/);
```

Comment:
It's obvious something needs to happen with `print`, but the exact change is unclear. Maybe make separate hint for it?